webpack.config.ts

```diff
@@ -26,13 +26,13 @@ var commonConfig = {
     // add 'var CompressionPlugin = require("compression-webpack-plugin");' on the top
     // and comment out below codes
     //
-    // new CompressionPlugin({
-    //   asset: "[path].gz[query]",
-    //   algorithm: "gzip",
-    //   test: /\.js$|\.css$|\.html$/,
-    //   threshold: 10240,
-    //   minRatio: 0.8
-    // })
+    // , new CompressionPlugin({
```

Comment:
@hearsid can you place a trailing comma at the end of all plugins. This is valid es5 code